server/routes/blackjack.py

```diff
@@ -32,16 +37,42 @@ def play_blackjack():
     except json.decoder.JSONDecodeError:
         return {"error": "Malformed request"}, 400
 
-@blackjack_bp.route("/api/blackjack/bet", methods=["POST"])
+@blackjack_bp.route("/api/blackjack/start", methods=["GET"])
 def bet_blackjack():
     try:
-        deck = get_deck_set()
+        # deck = get_deck_set()
```

Comment:
Don't leave this in the codebase if you aren't using it. It's in the git log anyway.